### PR TITLE
Strata - UI: Strata dashboard list

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity#24313

*Description of changes:*
- strata dashboard list/table
- choose account page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the strr2 license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-285-mfuh9g2p.web.app

File: strr-strata-web/app/app.config.ts [hunk 1]
```diff
@@ -1,3 +1,14 @@
 export default defineAppConfig({
+  strrBaseLayer: {
+    page: {
+      login: {
+        redirectPath: '/auth/account/choose-existing'
+        // options: { // allow all options?
```
Comment (kialj876) [inline strr-strata-web/app/app.config.ts:6]: good question I think we should allow everything for now

File: strr-strata-web/app/stores/strata.ts [hunk 2]
```diff
@@ -52,21 +52,48 @@ export const useStrrStrataStore = defineStore('strr/strata', () => {
...
+        return undefined
+      })
+      .then((response) => {
+        if (response) {
+          return (response as StrataApplicationResp[]).filter(
+            item => item.registration.registrationType === ApplicationType.STRATA_HOTEL
+          ).map(app => ({
+            hotelName: app.registration.strataHotelDetails.brand.name,
+            number: app.header.applicationNumber,
+            type: 'Application',
```
Comment (kialj876) [inline strr-strata-web/app/stores/strata.ts:88]: probably would be good to have this and 'Registration' come from i18n?

File: strr-strata-web/app/stores/strata.ts [hunk 3]
```diff
@@ -52,21 +52,48 @@ export const useStrrStrataStore = defineStore('strr/strata', () => {
...
+            expiryDate: '-',
+            status: app.header.status
+          }))
+        }
+        return []
+      })
```
Comment (kialj876) [inline strr-strata-web/app/stores/strata.ts:94]: nice this is clean

**Review by kialj876 — APPROVED** [review]
Nice work 👍

File: strr-strata-web/app/stores/strata.ts [hunk 4]
```diff
@@ -52,21 +52,48 @@ export const useStrrStrataStore = defineStore('strr/strata', () => {
...
+        logFetchError(e, 'Unable to load account registrations')
+        return undefined
+      })
+      .then((response) => {
+        if (response) {
+          return (response as StrataRegistrationResp[]).filter(
+            item => item.registrationType === ApplicationType.STRATA_HOTEL
+          ).map(reg => ({
+            hotelName: reg.strataHotelDetails.brand.name,
+            number: reg.registration_number,
```
Comment (kialj876) [inline strr-strata-web/app/stores/strata.ts:67]: Oh wait a minute can you just double check that the 'registration_number' works via postman to grab a specific registration? We might need the 'id' of the registrationwhen they click 'View', I'm not totally sure

Comment (deetz99) [reply strr-strata-web/app/stores/strata.ts:67]: Yeah 1 min

Comment (deetz99) [reply strr-strata-web/app/stores/strata.ts:67]: Yeah I'll need to use id, good catch

Comment (deetz99) [reply strr-strata-web/app/stores/strata.ts:69]: fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-285-mfuh9g2p.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-285-mfuh9g2p.web.app

**Review by kialj876 — APPROVED** [review]

**Pull request merged**